components/main-chef-wrapper/main.go

```diff
@@ -43,14 +46,24 @@ func main() {
 		cmd        *exec.Cmd
 	)
 
-	switch subCommand {
+	switch getAction(subCommand) {
 	case "report", "capture":
 		cmd = exec.Command(dist.AnalyzeExec, allArgs...)
 
+	case "policy-rollout":
+		cmd = exec.Command(dist.WorkstationExec, allArgs...)
+		runCmd(cmd)
+		allArgs = []string{"report-new-rollout", "-g", allArgs[1], "-l", allArgs[2],
+			"-s", os.Getenv("CHEF_AC_SERVER_URL"), "-u", os.Getenv("CHEF_AC_SERVER_USER")}
+		cmd = exec.Command(dist.AutomateCollectExec, allArgs...)
```

Comment:
For future iterations, I wonder if we can do this more transactionally - first, create the rollout record successfully in automate, then update the results after we execute the command.  This way even if the final update fails, we're guaranteed to have _some_ record of the rollout before we proceed to push the policy.